docs/resources/compute_keypair.md

```diff
@@ -4,6 +4,8 @@ subcategory: "Elastic Cloud Server (ECS)"
 
 # huaweicloud_compute_keypair
 
+-> **DEPRECATED:**  This resource  has been deprecated. Please use [huaweicloud_kps_keypair](https://www.terraform.io/docs/providers/huaweicloud/r/kps_keypair).
```

Comment:
should the `subcategory` be updated to **Deprecated** ?

BTW, this resource is also used in the usage of [cce_node](https://registry.terraform.io/providers/huaweicloud/huaweicloud/latest/docs/resources/cce_node#basic-usage).

Reply:
just keep here for a few releases before move to Deprecated. will replace the reference of this in the cce node scrpt.

Reply:
done